views/partials/compromised_accounts.hbs

```diff
@@ -13,5 +13,5 @@
       </div>
     </div>
   {{/each}}
-  <span id="source-info">Breach data provided by <span id="pwned">';--</span><span class="medium"> Have I been Pwned?</span></span>
+  <span class="source-info">Breach data provided by<span class="medium"> Have I been Pwned?</span></span>
```

Comment:
Looking at these side-by-side, is it OK that one of these HIPB citations is in a `<span class="demi" />` and the other is in a `<span class="medium" />`?

Reply:
Total oversight ! I'll fix this in next branch.